Update sidebar, landing page, and add new pages (#3899)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/jobs.md

---
title: "Jobs in dbt Cloud"
sidebar_label: "Jobs"
description: "Learn about deploy jobs and continuous integration (CI) jobs in dbt Cloud and what their differences are." 
tags: [scheduler]
---

In dbt Cloud, you can create and set up triggers for these jobs:
- [Deploy jobs](/docs/deploy/deploy-jobs)
- [Continuous integration (CI) jobs](/docs/deploy/continuous-integration)  

> :::tip Join our beta 

Reviewer: should this be a reusable? Just noticed it's slightly different than the [one above](https://github.com/dbt-labs/docs.getdbt.com/pull/3899/files#diff-b5cfa135eb077190c1e80ef584693ca77eb4c5cb8b0600390e1d66bd1764bb6eR20). But maybe that's intentional?

Author: i thought about that too. but each one is slightly different than the other. the one here needs to include both since the page talks about diff types of dbt cloud jobs (deploy jobs, ci jobs)

dbt Labs is currently running a beta that provides improved UI updates for setting up deploy jobs and CI jobs. For docs on deploy jobs, refer to [Create and schedule jobs (Beta version)](/docs/deploy/deploy-jobs?version=beta#create-and-schedule-jobs). For docs on CI jobs, refer to [Set up CI jobs (Beta version)](/docs/deploy/ci-jobs?version=beta#set-up-ci-jobs).

If you're interested in joining our beta, please fill out our Google Form to [sign up](https://forms.gle/VxwBD1xjzouE84EQ6).

:::

> Below is a comparison table that describes how deploy jobs and CI jobs behave differently:

Reviewer: So glad you added this table. It really really helps with understanding benefits/limitations of each.

|  | Deploy Jobs | CI Jobs |
| --- | --- | --- |

> | Purpose | Building production data assets | Building and testing new code before merging changes into production |

Reviewer: Just tiny nit to make the list parallel so it's easier to read
```suggestion
| Purpose | Builds production data assets | Builds and tests new code before merging changes into production |
```

| Trigger types | Triggered by a schedule or by API | Triggered by a webhook from a commit to a PR or by API |
| Destination | Builds into a production database and schema | Builds into a staging database and ephemeral schema, lived for the lifetime of the PR |
| Execution Mode | Runs execute sequentially, so as to not have collisions on the underlying DAG. | Runs execute in parallel to promote team velocity. |

> | Efficiency run savings | Over scheduled jobs are detected and unnecessary runs are canceled to avoid queue clog. | Run cancellation happens when an in-flight run becomes stale due to a new commit of the PR being pushed. |

Reviewer: Suggestions to make sentence active so it's easier to read for non native English speakers. Also writing out "pull request" in case people don't understand PR.

```suggestion
| Efficiency run savings | Detects over scheduled jobs and cancels unnecessary runs to avoid queue clog. | Cancels runs when an in-flight run becomes stale when a new commit is pushed to the pull request. |
```
